Jenkinsfile

```diff
@@ -107,8 +108,8 @@ def taskSourceCheckout(cfg) {
 def taskRunTest(cfg) {
 	cepl.cleanCurrentDir()
 	unstash(name: SOURCE_STASH)
-	papl.runCMakeBuild(SOURCE, 'build', 'build_and_run_tests', cfg, [])
-	junit('build/test/palladio_test_report.xml')
+	buildPalladio(cfg, [], 'build_and_run_tests')
+	papl.jpe.junit('build/test/palladio_test_report.xml')
```

Comment:
`papl.jpe` is the same as `this` in `Jenkinsfile` (injected via `cepl`) - you can leave it away.

There are additional occurrences of that below...